inventory/byo/hosts.ose.example

```diff
@@ -641,9 +641,9 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # See https://github.com/openshift/openshift-ansible/tree/release-3.6/roles/openshift_logging
 openshift_logging_es_cluster_size=1
 openshift_logging_es_memory_limit=4G
-openshift_logging_es_nodeselector='purpose=infra'
-openshift_logging_curator_nodeselector='purpose=infra'
-openshift_logging_kibana_nodeselector='purpose=infra'
+openshift_logging_es_nodeselector="{'region':'infra'}"
```

Comment:
These should match `openshift_logging_es_nodeselector={'region':'infra'}`, it shouldn't have surrounding double quotes

Reply:
Thanks